src/components/Player/GridControls.js

```diff
@@ -132,6 +132,12 @@ export default class GridControls extends Component {
       ']': 'forward',
     };
 
+    if (shiftKey) {
+      const isAcross = this.props.direction === 'across'
```

Comment:
nice, this is a clever way to get this feature working!

The way i had originally set up the key down handler didn't really scale (it passed in `shiftKey` into handleAction, which ends up being a bit clunky for cases other than tab), perhaps a refactor is in order to make it more extensible in the future.